recipes/sompy/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "sompy" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "846626a39e27cca171a81f4388cb30a32cdcd3bedf170c91227cf5a958aaabec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - sompy
+
+about:
+  home: https://github.com/ttlg/sompy
+  license: MIT License
```

Comment:
`license: MIT`
  
Also, please manually package the `license_file` and submit an issue/PR upstream, adding the url to the issue as a comment in the recipe. 

https://conda-forge.org/docs/meta.html#packaging-the-license-manually